quna/src/main/java/com/ktpt/quna/infra/LoginMemberIdInterceptor.java

```diff
@@ -0,0 +1,67 @@
+package com.ktpt.quna.infra;
+
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.Optional;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.method.HandlerMethod;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+import com.ktpt.quna.application.exception.InvalidTokenException;
+import com.ktpt.quna.domain.model.MemberRepository;
+import com.ktpt.quna.infra.annotation.LoginMemberId;
+import com.ktpt.quna.infra.token.JwtTokenProvider;
+import com.ktpt.quna.infra.token.TokenExtractor;
+
+@Component
+public class LoginMemberIdInterceptor implements HandlerInterceptor {
+	private final JwtTokenProvider jwtTokenProvider;
```

Comment:
```suggestion
    private final JwtTokenProvider jwtTokenProvider;
```

일부 파일에서 tab 사용한걸로 나오네요!

Reply:
![image](https://user-images.githubusercontent.com/37579660/105250148-9bc09180-5bbc-11eb-8dd2-c4edf2801e96.png)
자동완성에서 포매팅을 할 때에 확인을 해보니 계속 걸리더라구요. 
4space를 해놓은 줄 알았는데 탭을 사용하고 있었나봅니다.
해당 부분을 경준님 pr에서도 수정해주셨으나 저도 수정해서 처리할게요~

Reply:
![image](https://user-images.githubusercontent.com/37579660/105250810-b5aea400-5bbd-11eb-8f2f-20f88068782d.png)
요 부분이 저와 컨벤션이 조금 다른 것 같아요..!
tab관련한 부분도 우선은 커밋 안해두겠습니다. 만나게 되면 처리하죠!